iohk/iohk-ops.hs

```diff
@@ -181,6 +182,8 @@ centralCommandParser =
    , ("destroy",                "Destroy the whole cluster",                                        pure Destroy)
    , ("delete",                 "Unregistr the cluster from NixOps",                                pure Delete)
    , ("fromscratch",            "Destroy, Delete, Create, Deploy",                                  pure FromScratch)
+   , ("reallocate-core-ips",    "Destroy elastic IPs corresponding to the nodes listed and reprovision cluster",
```

Comment:
Reprovision means that resources would be reprovisioned, we should just say "deploy"

Reply:
Done.